lib/errors/course_update_error_handling.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module CourseUpdateErrorHandling
```

Comment:
Maybe call this `UpdateServiceErrorLogging` This doesn't do any actual handling of errors, in terms of flow control... its only job is to make sure the errors get logged appropriately.

Reply:
Done!